libs/wingc/src/type_check.rs

```diff
@@ -190,6 +190,9 @@ pub struct Class {
 	pub is_abstract: bool,
 	pub type_parameters: Option<Vec<TypeRef>>,
 	pub phase: Phase,
+
+	/// The phase in which this class was declared
+	pub declaration_phase: Phase,
```

Comment:
Note that this should be identical to the `phase` field of the `env` in which the lookup found the type. See `SymbolLookupInfo`. I tried to add some documentation in symbol_env.rs to clear this up, and hopefully this can be used instead of of adding this field here.

Reply:
I am removing this